<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MTL Trigger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0,0,0">
    <item row="15" column="1" colspan="7">
     <widget class="QGroupBox" name="trigger_properties">
      <property name="title">
       <string>Trigger properties</string>
      </property>
      <layout class="QFormLayout" name="trigger_properties_layout"/>
     </widget>
    </item>
    <item row="10" column="6" colspan="2">
     <widget class="QCheckBox" name="ignorepowerstatus_chb">
      <property name="text">
       <string>Ignore Power Status</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="7">
     <widget class="QTableView" name="data_view">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="trigger_label">
      <property name="text">
       <string>Trigger Type:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="5">
     <widget class="QPushButton" name="connect_btn">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QFrame" name="state_frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="3">
     <widget class="QLineEdit" name="station_le"/>
    </item>
    <item row="13" column="2" colspan="2">
     <widget class="QComboBox" name="trigger_type"/>
    </item>
    <item row="13" column="5">
     <widget class="QPushButton" name="trigger_switch">
      <property name="styleSheet">
       <string notr="true">QPushButton:checked { background-color: rgb(255, 0, 0); }</string>
      </property>
      <property name="text">
       <string>Trigger On</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
